### PR TITLE
Update docs to use xargs to process sites individually

### PR DESCRIPTION
**Pull request by nathanielks**
Our internal Cloud docs have recommended this for a minute so I wanted to make sure our user facing docs reflect this!

### CODE REVIEW

**Comment by roborourke** [comment]
The `--network-wide` flag has other effects we should take into account such as creating the network wide search alias. I'd like to double check that this won't miss anything out that the plugin could be expecting to be present.

**Comment by nathanielks** [comment]
@roborourke is that something you have the bandwidth for or do need help with that?

**Comment by nathanielks** [comment]
Rather, if it's at all possible for you to investigate that further, it would be a significant help to me, otherwise I'm happy to do the research!

File: docs/reindexing-content.md [hunk 1]
```diff
@@ -6,24 +6,26 @@ When making changes the data or analyzers that are stored in the search index, y
...
 
 ```sh
-wp elasticpress index --network-wide
+wp site list --field=url | xargs -I % wp elasticpress index --url=%
 ```
 
-Re-sychronise the index and updating mappings (needed when updating synonym definitions)
+`wp site list --field=url` will print each site URL on a new line and ingested by `xargs` to be processed individually.
+
+To re-sychronise the index and update mappings (needed when updating synonym definitions):
```
Comment (roborourke) [inline docs/reindexing-content.md:17]:
```suggestion
To re-sychronise the index and update mappings (needed when the mapping has been modified via filters, site language has been changed or a user dictionary has been added and not reindexed successfully):
```

File: docs/reindexing-content.md [hunk 2]
```diff
@@ -6,24 +6,26 @@ When making changes the data or analyzers that are stored in the search index, y
...
 
 ```sh
-wp elasticpress index --setup
+wp site list --field=url | xargs -I % wp elasticpress index --url=% --setup
```
Comment (roborourke) [inline docs/reindexing-content.md:20]:
```suggestion
wp site list --field=url | xargs -I % wp elasticpress index --url=% --setup && wp elasticpress recreate-network-alias
```

**Review by roborourke — CHANGES_REQUESTED** [review]
When doing the set up without the `--network-wide` flag the main thing missed is the network alias creation, there is a CLi command that can be tacked on to handle this though. Added as a suggestion but not sure if `xargs` will handle the `&&` as added here.

File: docs/reindexing-content.md (hunk 2 above)
Comment (nathanielks) [reply docs/reindexing-content.md]: Does this need to be run once or for each network? If only once, your recommendation is sound! Bash picks up the `&&` before `xargs` does, so it's treated as a separate command from `xargs`.

Comment (roborourke) [reply docs/reindexing-content.md]: Ah that's what I was worried about, needs to be scoped to the xargs command

Comment (nathanielks) [reply docs/reindexing-content.md]: Looking at the source, it appears to loop through all sites, so I believe this should run outside of the xargs command (how this PR has it currently written). What do you think?

2.8.2: https://github.com/10up/ElasticPress/blob/b56e8f8dc2f07cd6c591edc5db94eb49236bc549/bin/wp-cli.php#L282-L310

latest: https://github.com/10up/ElasticPress/blob/c8c686d49f588d8ccfdc4e1cf0b22d9459f8276e/includes/classes/Command.php#L456-L486

Comment (roborourke) [reply docs/reindexing-content.md]: Missed that bit! Alrighty then, we're good here

**Review by roborourke — APPROVED** [review]

**Pull request merged**